### PR TITLE
Removed transcript files and git ignored them.

### PR DESCRIPTION
**Pull request by GustavoCig**
Description:Added extra lines in the .gitignore file to git ignore all transcripts.
Commit:Rem
TaskNumber:#22

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,6 +2,7 @@ Teste/Out/*.txt
 Teste/In/*.txt
 Teste/work/
 Teste/transcript
+Teste/NoC/transcript
```
Comment (jmarcelof) [inline .gitignore]: I think you could replace these lines by "*transcript".

**Comment by jmarcelof** [comment]
Please 'fixup' the commits together and I'll merge it.

**Comment by GustavoCig** [comment]
It ended commiting twice for some reason, but is this what you meant by 'fixup'?

**Comment by jmarcelof** [comment]
Take a look at this: https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History#Squashing-Commits

**Comment by GustavoCig** [comment]
I keep both commit messages after using 'squash' or unify them in a single one?

**Comment by jmarcelof** [comment]
Unify them (using fixup mode on 'rebase -i' will keep only the first message).

**Pull request merged**